src/app/modules/domain/knowledge-component/knowledge-component.component.ts

```diff
@@ -65,4 +72,25 @@ export class KnowledgeComponentComponent implements OnInit {
       this.learningObjects[0] = assessmentEvent;
     });
   }
+
+  private registerEventListeners(): void {
+    this.knowledgeComponentService.submitEvent.subscribe(value => {
+      {
+        this.aeCorrectnessLevel = value;
+        this.aeSubmitted = true;
+      }
+    });
+
+    this.knowledgeComponentService.nextPageEvent.subscribe(value => {
+      {
+        this.aeCorrectnessLevel = 0.0;
+        this.aeSubmitted = false;
+        if (value === 'AE') {
```

Comment:
We should change the UI to be in Serbian and give a more descriptive name. E.g., "Sledeći zadatak" and "Pregled gradiva".